examples/product_kernel.jl

```diff
@@ -4,7 +4,7 @@ using DifferentialEquations
 using Plots
 
```

Comment:
Should we also add examples to Travis?

Reply:
Anyhow. The examples don't work for me due to some plotting issues:
```Warning: PyPlot is using tkagg backend, which is known to cause crashes on MacOS (#410); use the MPLBACKEND environment variable to request a different backend.
└ @ PyPlot ~/.julia/packages/PyPlot/4wzW1/src/init.jl:192
ERROR: LoadError: MethodError: no method matching _show(::IOStream, ::MIME{Symbol("image/png")}, ::Plots.Plot{Plots.PyPlotBackend})
The applicable method may be too new: running in world age 25684, while current world is 25772.
Closest candidates are:
  _show(::IO, ::MIME{Symbol("image/png")}, ::Plots.Plot{Plots.PyPlotBackend}) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/backends/pyplot.jl:1399 (method too new to be called from this world context.)
  _show(::IO, !Matched::MIME{Symbol("text/html")}, ::Plots.Plot) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:164
  _show(::IO, !Matched::MIME{Symbol("text/plain")}, ::Plots.Plot) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:208
  ...
Stacktrace:
 [1] show(::IOStream, ::MIME{Symbol("image/png")}, ::Plots.Plot{Plots.PyPlotBackend}) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:199
 [2] png(::Plots.Plot{Plots.PyPlotBackend}, ::String) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:8
 [3] savefig(::Plots.Plot{Plots.PyPlotBackend}, ::String) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:122
 [4] savefig(::String) at /Users/ajaruga/.julia/packages/Plots/2KhB2/src/output.jl:124
 [5] main() at /Users/ajaruga/clones/cloudy/examples/product_kernel.jl:59
 [6] top-level scope at none:0
 [7] include at ./boot.jl:326 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] exec_options(::Base.JLOptions) at ./client.jl:267
 [11] _start() at ./client.jl:436
in expression starting at /Users/ajaruga/clones/cloudy/examples/product_kernel.jl:62
```

Reply:
After some googling I found this: https://github.com/JuliaPlots/Plots.jl/issues/745

Following the discussion there adding ```gr()``` after ```pyplot()``` solves my problem.